modules/submit-score.js

```diff
@@ -0,0 +1,31 @@
+// submit-score.js
+import API_BASE_URL from './config.js';
+
+function handleSubmissionError(error) {
+  // Perform custom error handling, e.g., display an error message on the webpage
+  const errorMessage = document.querySelector('#submission-error');
+  errorMessage.textContent = `Error submitting score: ${error.message}`;
+  errorMessage.style.display = 'block'; // Show the error message
+}
+
+async function submitScore(gameId, userName, score) {
```

Comment:
- [ ] It would be better if we use the `ES6 arrow syntax` instead of the `function` keyword